frontend/src/modules/editor/selectTheme/views/editorThemeItemView.js

```diff
@@ -24,6 +24,19 @@ define(function(require){
     },
 
     postRender: function() {
+      var previewUrl = '/api/theme/preview/' + this.model.get('name') + '/' + this.model.get('name')
+      var $previewLoc = this.$('.theme-preview');
+
+      $.ajax(previewUrl, {
+        statusCode: {
+          200: function() {
+            $previewLoc.prepend($('<img/>', { src: previewUrl, alt: Origin.l10n.t('app.themepreviewalt') }));
+          },
+          204: function() {
+            $previewLoc.prepend($('<i/>', { class: 'fa fa-file-image-o' }));
```

Comment:
Very minor suggestion: I wonder if we can use something more theme-related for the icon here, as we use the image file one in various other places in the app's asset management. Best font awesome icon I can find is the paintbrush (there may be something better):

![screen shot 2017-08-21 at 11 39 39](https://user-images.githubusercontent.com/1059083/29515758-93b71e94-8665-11e7-9a22-d29cdb609bd6.png)